.gitignore

```diff
@@ -84,7 +84,6 @@ tests/cases/user/*/**/*.d.ts
 tests/baselines/reference/dt
 .failed-tests
 TEST-results.xml
-package-lock.json

```

Comment:
This should stay, so local changes (if there are any) aren't added by a `git add .`. The auto-update script should forcibly add the lockfile despite the ignore.

Reply:
Great idea!